test/mod/test_host.py

```diff
@@ -135,6 +145,42 @@ def test_exception():
             client.call("exception")
 
 
+def test_signals():
+    with host.ServiceManager() as mgr:
+        exec_callback = False
+        # pylint: disable=unused-argument
```

Comment:
You can prefix variables with `_` and pylint won't warn about them not being used anymore.